### ISSUE TITLE
Quick-hop command "ctrl+shift+arrow" resets chat configs

### ISSUE THREAD

**Issue by MalakaJones** [runelite/runelite#5299]
After performing a quick-hop using that key shortcut, my chatbox configs gets reset.

![image](https://user-images.githubusercontent.com/15826583/45048562-3d75d100-b04a-11e8-8e4f-a5b8b428505d.png)
![image](https://user-images.githubusercontent.com/15826583/45048586-4b2b5680-b04a-11e8-8cca-fb0884734d30.png)

**Comment by Unmoon** [comment]
This is a Jagex bug.

**Comment by MalakaJones** [comment]
You're right, I've never realized using the in-game world hopping system does the same thing.

**Comment by Unmoon** [comment]
I think the game just doesn't save Game/Clan option when you log out.

**Comment by MalakaJones** [comment]
I'm pretty sure it used to.
